docs/GRAPHQL.md

```diff
@@ -1,5 +1,14 @@
 # graphql info/spec
 
+## backwards-compatibility notes
+
+the following features of the subgraph graphql api are explicitly unsupported, because they are not used by ensjs or ens-app-v3
+
+- `_nocase` case-insensitive where filters
+- [fulltext search queries](https://thegraph.com/docs/en/subgraphs/querying/graphql-api/#fulltext-search-queries)
+- 1-level-nested Entity `_orderBy` param
+- subgraph `_Meta_` object (ponder's `_meta` is available)
```

Comment:
Thank you for an explicit list of API elements not utilised by the clients we seek to support.